contracts/eth/nearbridge/contracts/NearDecoder.sol

```diff
@@ -7,6 +7,9 @@ library NearDecoder {
     using Borsh for Borsh.Data;
     using NearDecoder for Borsh.Data;
 
+    uint8 constant VALIDATOR_V1 = 1;
```

Comment:
As I see this variable is not used, could we avoid that?

Reply:
I'd prefer to keep the constant for the visibility so it will be easier to remember later how these constants for versions are produced and also that `VALIDATOR_V2` value is not just a random number.